xml/docu_styleguide.smartdocs.xml

```diff
@@ -337,4 +337,58 @@ ultrashort description for social media, max. 55 chars&lt;/meta>
    </tgroup>
  </informaltable>
 </sect2>
+
+<sect2 xml:id="sec-revhistory">
+  <title>Revision history</title>
+  <para>Update revision history regularly, mentioning most important changes to
+  your document when you amend or rework it. The <quote>Revision History</quote> 
+  changelog is available as a link before the abstract and opens up in its individual
+  page. Keep in mind the following rules:</para>
+  <itemizedlist>
+    <listitem>
+     <para>Describe only significant changes that you performed.</para>
+    </listitem>
+    <listitem>
+     <para>If you have several changes of the same type (addition, change, removal),
+      group them under a dedicated <tag class="emptytag">itemizedlist</tag>.</para>
+      </listitem>
+      <listitem>
+        <para>Do not add links of any kind.</para>
+      </listitem>
+      </itemizedlist>
+      <example xml:id="ex-revhistory-source">
+        <title>Revision history example (source)</title>
+<screen>
+&lt;revision&gt;&lt;date&gt;2024-11-13&lt;/date&gt;
+  &lt;revdescription&gt;
+    &lt;itemizedlist&gt;  
+     &lt;listitem&gt;&lt;para&gt;Added sections to resolve issue 12345:&lt;/para&gt;&lt;/listitem&gt;
```

Comment:
"Issue 12345" can be located anywhere and is not very specific. If readers wants to look into it, they are lost and don't know where it is (Bugzilla? GitHub? GitLab? Anything else?). 

For this reason, I would highly recommend to be a little bit more specific (or don't mention the issue at all). Therefor, I'd suggest to introduce the abbreviation convention that I describe above (bsc for our Bugilla tracker, gh for GitHub etc.)

The improved example would look like:

```suggestion
     &lt;listitem&gt;&lt;para&gt;Added sections to resolve issue &lt;uri>bsc#12345&lt;/uri>:&lt;/para&gt;&lt;/listitem&gt;
```